lib/common/output.c

```diff
@@ -168,6 +170,12 @@ pcmk__register_messages(pcmk__output_t *out, const pcmk__message_entry_t *table)
     }
 }
 
+void
+pcmk__register_common_messages(pcmk__output_t *out)
+{
+    pcmk__register_patchset_messages(out);
+}
```

Comment:
I wonder whether it would be worthwhile to just do this in `pcmk__output_new()`

Reply:
That sounds reasonable